public/modules/base/views/signup.html

```diff
@@ -73,6 +73,21 @@
                     </p>
                 </div>
                 
+                <div class="form-group"
+                     ng-class="{
+                     'has-warning': form.username.$dirty,
+                     'has-success': form.username.$valid && form.username.$dirty && !form.username.$error.mongoose,
```

Comment:
we have are using the mongose errors in order to detect and show errors, I think this can be improved decoupling that created new ticket: https://github.com/MakingSense/mean-seed/issues/30 @cwilgenhoff-ms could youtry to improve this? maybe this directive can help: https://github.com/nelsonomuto/angular-ui-form-validation